conanfile.py

```diff
@@ -45,6 +43,13 @@ def package(self):
         """
         self.copy(pattern="LICENSE.txt", dst="licenses")
         self.copy(pattern="flathash*", dst="bin", src="bin")
+        if self.settings.os == "Windows" and self.options.shared:
+            if self.settings.compiler == "Visual Studio":
+                shutil.move(os.path.join(self.package_folder, "lib", "%s.dll" % self.name),
```

Comment:
Why these extra ``shutil.move``? Why not standard ``self.copy()``?
Also, why necessary if there is already a ``cmake.install()``?

Reply:
flatbuffers installs runtime libraries on `lib` folder. After to install all libraries, I moved from lib to bin. Otherwise, will be necessary a workaround on `package_info`.

Reply:
Oh, I see, thanks for the explanation, it is ok then.